tests/phpunit/tests/rest-api/rest-search-controller.php

```diff
@@ -332,6 +352,28 @@ public function test_get_items_search_for_foocontent() {
 		);
 	}
 
+	/**
+	 * @ticket 55674
+	 * Make sure post ids are primed.
+	 */
+	public function test_get_items_search_prime_ids() {
+		global $wpdb;
+
+		$query_args = array(
+			'per_page' => 100,
+			'search'   => 'primedpoststitles',
+		);
+		$response   = $this->do_request_with_params( $query_args );
+		$this->assertSame( 200, $response->get_status() );
```

Comment:
Tests with multiple asserts should really have a message added. See https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions